docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.9"
+
+x-common-env: &config_env
+  PROP:
+
+services:
+  app:
```

Comment:
🤔 Pensando se vale já deixar setado o volume (mapear o local com o dentro do container), sabe?

Mas eu tmb só vejo utilidade se tiver um live reload (não consegui então larguei mão)
Sabe se tem? Se é fácil?

Se sim, vale? E, se sim, nesse momento?



Eu cheguei a testar mas não consegui. Sabe se tem como ter um live reload?

Reply:
Pode explicar melhor o lance do live reload e como ele se relaciona com o Volume? Não conseguimos entender direito

Reply:
Não sei se usei os termos certo hauhaua
- Mas basicamente o q eu to querendo é poupar umas etapas de dev

<br/>

#### Live reload
Eu to falando do reload, rebuild automático quando tem alteração de código
- É bem comum em aplicações de frontend (ele recarrega automático)
- O Django faz isso por padrão, dá pra configurar pra não fazer

Eu sei que exisrte o termo "hot reload" e sei que tem diferenças entre eles, mas qual é e qual o melhor aí eu nunca tenho ctz ahuahuah
<br/>

#### Volume
Eu falei volume mas tava falando do [bind](https://docs.docker.com/storage/bind-mounts/)

De forma simplificada, ele referencia / copia o diretório do seu sistema (src) pra dentro do container (dest). 
- Por default, quando altera na máquina precisa dar um rebuild, reup do container pra atualizar
- Com o bind, essa atualização de arquivos é feita automática
<br/>

#### Como se relacionam
- Se tiver o bind, vai atualizar automático dentro do container
- Nisso, o reload vai ser disparado (pq, dentro do container, o código vai ter sido alterado)

---

Deu de entender? Fez sentido? Too much?

Reply:
Acho que seria uma boa discutir com os outros isso também
Por enquanto acho que vou criar uma tarefa separada com esses pontos, o que acha? Aí viria como uma melhoria e liberaria esse PR com um docker compose funcional básico

Reply:
Parfait